db/migrate/20230824091929_add_user_id_to_reports.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserIdToReports < ActiveRecord::Migration[7.0]
+
+  def change
+    add_reference :reports,:user, foreign_key: true
+    change_column :reports, :user_id,:integer,null: false
```

Comment:
```suggestion
    change_column :reports, :user_id, :integer, null: false
```

スペースが抜けてます。（rubocopを実行してない？）

Reply:
rubcopを再度実行してみましたが、修正されませんでした。
目視での確認も怠らないようにします。

![スクリーンショット 2023-09-06 9 48 27](https://github.com/jasmine-tokudome/fjord-books_app-2023/assets/103996128/f7452c26-1286-497a-9abb-91bd8020f54c)

Reply:
あ、すいません、migrationファイルはチェックしない設定になってました。
（自動生成されたコードはrubocopのルールに沿ってないケースがあるため）
https://github.com/fjordllc/rubocop-fjord/blob/master/config/rails.yml#L12

ということは、開発者が気を付ける必要がありますね。

Reply:
ご確認いただきありがとうございました:pray: